### ISSUE TITLE
Sensible conversion for `aten.sym_size.int`

### ISSUE THREAD

**Issue by jdh8** [tenstorrent/pytorch2.0_ttnn#185]
[`aten.sym_size.int`](https://pytorch.org/docs/stable/torch.compiler_ir.html) returns a `SymInt` representing the size at the given dimension.  This op does not return a tensor.  What should we do with this op?

> ### aten.sym_size.int
> |    | ATen Input Variations                                   | Status   | 
> |---:|:--------------------------------------------------------|:---------|
> |  0 | Tensor<[1, 1, 1, s10 + 1]> self = ?,<br>int<> dim = 3   | Unknown  |
> |  1 | Tensor<[1, 12, s0 + 1, 64]> self = ?,<br>int<> dim = 2  | Unknown  |
> |  2 | Tensor<[1, 12, s10 + 1, 64]> self = ?,<br>int<> dim = 2 | Unknown  |
> |  3 | Tensor<[1, 16, s0 + 1, 64]> self = ?,<br>int<> dim = 2  | Unknown  |
> |  4 | Tensor<[1, 16, s10 + 1, 64]> self = ?,<br>int<> dim = 2 | Unknown  |
> |  5 | Tensor<[1, 6, s0 + 1, 64]> self = ?,<br>int<> dim = 2   | Unknown  |
> |  6 | Tensor<[1, 8, s0 + 1, 64]> self = ?,<br>int<> dim = 2   | Unknown  |
> |  7 | Tensor<[1, s0, 256]> self = ?,<br>int<> dim = 1         | Unknown  |
> |  8 | Tensor<[1, s0, 768]> self = ?,<br>int<> dim = 1         | Unknown  |
> |  9 | Tensor<[1, s10 + 1]> self = ?,<br>int<> dim = 1         | Unknown  |
>
> https://github.com/tenstorrent/pytorch2.0_ttnn/blob/main/docs/cumulative_input_variations.md#atensym_sizeint

**Comment by jerrysky3** [comment]
I'm also running into this when lowering `aten.nonzero`

Compile:

```py
class NonzeroModule(torch.nn.Module):
    def __init__(self):
        super().__init__()

    def forward(self, input, as_tuple):
        return torch.nonzero(input, as_tuple=as_tuple)
```

Generate:

```py
ExportedProgram:
    class GraphModule(torch.nn.Module):
        def forward(self, l_input_: "bf16[16]", arg1):
            # File: /home/jerry/pytorch2.0_ttnn/tests/lowering/data_movement/test_nonzero.py:12, code: return torch.nonzero(input, as_tuple=as_tuple)
            nonzero: "i64[i1, 1]" = torch.ops.aten.nonzero.default(l_input_);  l_input_ = None

            # File: /home/jerry/pytorch2.0_ttnn/venv/lib/python3.8/site-packages/torch/_export/pass_base.py:44, code: return NodeMetadata({"stack_trace": "".join(traceback.format_stack(limit=1))})
            sym_size: "Sym(i1)" = torch.ops.aten.sym_size.int(nonzero, 0)
            ge: "Sym(i1 >= 0)" = sym_size >= 0
            scalar_tensor: "f32[]" = torch.ops.aten.scalar_tensor.default(ge);  ge = None
            _assert_async = torch.ops.aten._assert_async.msg(scalar_tensor, 'nonzero.shape[0] is outside of inline constraint [0, 16].');  scalar_tensor = None
            le: "Sym(i1 <= 16)" = sym_size <= 16;  sym_size = None
            scalar_tensor_1: "f32[]" = torch.ops.aten.scalar_tensor.default(le);  le = None
            _assert_async_1 = torch.ops.aten._assert_async.msg(scalar_tensor_1, 'nonzero.shape[0] is outside of inline constraint [0, 16].');  scalar_tensor_1 = None
            return (nonzero,)
```

Since it is trying to get the actual shape of a tensor at runtime, can we define a custom `torch.fx.wrap` function to call in fx graph and return the size of a tensor at runtime? I'm thinking something like:

```py
@torch.fx.wrap
def get_size(t, dim):
    return t.shape[dim]
```
